docs/components/PkgLayout.tsx

```diff
@@ -51,19 +50,11 @@ export function PkgLayout({
 					)
 				}
 			/>
-			{isUnreleased ? (
+			{/* {isUnreleased ? (
 				<PageAlert tone="warning">
 					<Text as="p">This package is yet to be released on npm.</Text>
 				</PageAlert>
-			) : (
-				<Prose>
```

Comment:
This code has been moved into `pages/components.code.tsx`

Rather than showing how to import the package, we now show how to import the components


```
import { ... } from '@ag.ds-next/react/box';
```